diff --git a/build/index.js b/build/index.js
index 24f24af4c0624afb5605b1cf3d430e4ce240a713..5d0c41a9b8b80f9af084349664748621db25e7fb 100644
--- a/build/index.js
+++ b/build/index.js
@@ -1,9 +1,7 @@
-import { EventEmitter } from "expo-modules-core";
 const ExpoQuickActions = globalThis.expo?.modules
     ?.ExpoQuickActions;
 export const { initial, maxCount, setItems, isSupported } = ExpoQuickActions;
-const emitter = new EventEmitter(ExpoQuickActions);
 export function addListener(listener) {
-    return emitter.addListener("onQuickAction", listener);
+    return ExpoQuickActions.addListener("onQuickAction", listener);
 }
 //# sourceMappingURL=index.js.map
\ No newline at end of file
